models/payment/src/payment-method-info/payment-method-info-draft/transformers.ts

```diff
@@ -0,0 +1,28 @@
+import { Transformer } from '@commercetools-test-data/core';
+import type {
+  TPaymentMethodInfoDraft,
+  TPaymentMethodInfoDraftGraphQL,
+} from '../types';
+
+const transformers = {
+  default: Transformer<TPaymentMethodInfoDraft, TPaymentMethodInfoDraft>(
+    'default',
+    {
+      buildFields: ['name'],
+    }
+  ),
+  rest: Transformer<TPaymentMethodInfoDraft, TPaymentMethodInfoDraft>('rest', {
+    buildFields: ['name'],
+  }),
+  graphql: Transformer<TPaymentMethodInfoDraft, TPaymentMethodInfoDraftGraphQL>(
+    'graphql',
+    {
+      buildFields: ['name'],
```

Comment:
`name` is an array of type `LocalizedStringItemInputType`, so we should probably transform from the LocalizedField we build (the data's there, it's just not the write shape I think - correct me if I'm wrong)

![image](https://github.com/commercetools/test-data/assets/15024562/fc8739ee-da9f-4b43-bc3f-a9f43cd2b330)

![image](https://github.com/commercetools/test-data/assets/15024562/f69ff7cf-c9b9-44b1-bfab-7f12cee2b7c8)

Reply:
@jaikamat I haven't come across a transformation like this yet, is there an example you can think of? Or is it `LocalizedString.toLocalizedField(fields.name);`?

Reply:
@jaikamat It's genuinely unclear to me if we've ever run into this input type before. Can you think of an example?

Reply:
The test for name looks like:
```
        name: expect.arrayContaining([
          expect.objectContaining({
            locale: expect.any(String),
            value: expect.any(String),
          }),
        ]),
```
Which, looks right to me? Am I missing something?